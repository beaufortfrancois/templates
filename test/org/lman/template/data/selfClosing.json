{
  "t": true,
  "f": false,
  "foo": "foo value",
  "bar": "bar value",
  "value": 42,
  "list": [1, 2, 3],
  "object": {"x": 43},
  "objectList": [{"x": 1}, {"x": 2}, {"x": 3}]
}
