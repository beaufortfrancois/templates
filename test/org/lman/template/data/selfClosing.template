// Variables.
Some boolean values -{{?t value/}} -{{?f value/}} -{{?t object.x/}} -{{?f object.x/}}.
Inverted: -{{^t value/}} -{{^f value/}} -{{^t object.x/}} -{{^f object.x/}}.
Other conditions: -{{?value value/}} -{{^value value/}}.
List: -{{#list @/}} -{{#list value/}}.
Object: -{{#object x/}} -{{?object x/}} -{{^object x/}}.
Object list: {{#objectList x/}}.
// Partials.
Some partials:
-{{?t +partials.selfClosing_p1 /}} -{{?f +partials.selfClosing_p1 /}}
-{{^t +partials.selfClosing_p1 /}} -{{^f +partials.selfClosing_p1 /}}
// JSON.
Some JSON: -{{?t *list /}} -{{?f *list /}} -{{^t *list /}} -{{^f *list /}}
