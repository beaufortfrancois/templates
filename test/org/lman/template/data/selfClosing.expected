// Variables.
Some boolean values -42 - -43 -.
Inverted: - -42 - -43.
Other conditions: -42 -.
List: -123 -424242.
Object: -43 -43 -.
Object list: 123.
// Partials.
Some partials:
-This is a partial. -
- -This is a partial.
// JSON.
Some JSON: -[1,2,3] - - -[1,2,3]
